### PR TITLE
Woodapple Hotfix

### PR DESCRIPTION
**Pull request by Kurfursten**
Fixes the woodapple production problem by reverting it to a child of apple

Tweaked the vapor salt output (reduced)

### CODE REVIEW

File: code/modules/hydroponics/seeds.dm [hunk 1]
```diff
@@ -1676,7 +1676,7 @@
 	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/vaporsac)
 	plant_dmi = 'icons/obj/hydroponics2.dmi'
 	plant_icon = "vaporsac"
-	chems = list(VAPORSALT = list(1,1))
+	chems = list(VAPORSALT = list(1,2))
```
Comment (ComicIronic) [inline code/modules/hydroponics/seeds.dm:1679]: How is this a reduction - aren't you expanding the range of produced values?

Comment (Kurfursten) [inline code/modules/hydroponics/seeds.dm:1679]: No, the second value is how much to divide by

I know this is intuitive. Why else do you think there are about 3 people who understand plantcode?

**Comment by Kurfursten** [comment]
Special thanks to @Skullyton for pointing this out

Pls merge @ComicIronic

**Comment by Kurfursten** [comment]
@clusterfack @9600bauds @gbasood It admittedly feels a bit less hot of a fix after an hour

**Comment by JustSumBody** [comment]
yes hello I heard there were merges in here?
if so please merge my PR too I need the pomf coins for rent this month because I blew them all on antag rolls and Zero Time Dilemma and oh god my family is starving

**Comment by Kurfursten** [comment]
Truly we are bereft in an uncaring world without merges

**Comment by ghost** [comment]
Kurfurst, while you're here, how do you make the box cloner and box ranching kit etc. ?

**Comment by Kurfursten** [comment]
They're a premium item at the Vox Seed 'n' Feed - just insert a coin.

http://ss13.moe/wiki/index.php/Guide_to_Vox_Hydroponics

**Comment by Kurfursten** [comment]
Thanks CF

**Pull request merged**